modules/rosa-deleting-sts-iam-resources-account-wide.adoc

```diff
@@ -1,18 +1,58 @@
 // Module included in the following assemblies:
 //
 // * rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.adoc
+// *rosa_hcp/rosa-hcp-deleting-cluster.adoc
+
+ifeval::["{context}" == "rosa-hcp-deleting-cluster"]
+:hcp:
+endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="rosa-deleting-sts-resources-account-wide_{context}"]
 = Deleting the account-wide IAM resources
 
-After you have deleted all {product-title} (ROSA) with AWS Security Token Services (STS) clusters that depend on the account-wide AWS Identity and Access Management (IAM) resources, you can delete the account-wide resources.
+After you have deleted all 
+ifndef::hcp[]
+{product-title} (ROSA) with AWS Security Token Services (STS) 
+endif::hcp[]
+ifdef::hcp[]
+{hcp-title-first} 
+endif::hcp[]
+clusters that depend on the account-wide AWS Identity and Access Management (IAM) resources, you can delete the account-wide resources.
 
-If you no longer need to install a ROSA with STS cluster by using {cluster-manager-first}, you can also delete the {cluster-manager} and user IAM roles.
+If you no longer need to install a 
+ifndef::hcp[]
+ROSA with STS 
+endif::hcp[]
+ifdef::hcp[]
+{hcp-title} 
+endif::hcp[]
+cluster by using {cluster-manager-first}, you can also delete the {cluster-manager} and user IAM roles.
 
 [IMPORTANT]
 ====
-The account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the resources if they are not required by other clusters.
+The account-wide IAM roles and policies might be used by other 
+ifndef::hcp[]
+ROSA 
+endif::hcp[]
+ifdef::hcp[]
+{hcp-title} 
+endif::hcp[]
+clusters in the same AWS account. Only remove the resources if they are not required by other clusters.
 
-The {cluster-manager} and user IAM roles are required if you want to install, manage, and delete other ROSA clusters in the same AWS account by using {cluster-manager}. You must only remove the roles if you no longer need to install ROSA clusters in your account by using {cluster-manager}. See the "Additional resources" section for information on repairing your cluster if these roles are removed prior to deletion.
-====
+The {cluster-manager} and user IAM roles are required if you want to install, manage, and delete other 
+ifndef::hcp[]
+ROSA 
+endif::hcp[]
+ifdef::hcp[]
+{product-title} 
+endif::hcp[]
+clusters in the same AWS account by using {cluster-manager}. Only remove the roles if you no longer need to install 
+ifndef::hcp[]
+ROSA 
+endif::hcp[]
+ifdef::hcp[]
+{product-title} 
+endif::hcp[]
+clusters in your account by using {cluster-manager}. See the "Additional resources" section for information on repairing your cluster if these roles are removed before deletion.
```

Comment:
Here, I would just say
```suggestion
clusters in your account by using {cluster-manager}. For more information about repairing your cluster if these roles are removed before deletion, see "Repairing a cluster that cannot be deleted" in _Troubleshooting cluster deployments_.
```

Then do a trailing include, `.Additional resources` section right below this module in the assembly. I made a note in the assembly.

There is a new section in the repo docs that goes over how to handle these situations with a lot of great detail:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences

I do need to raise an issue with the team about the section on xrefs, though. In the additional resources section, we are not supposed to use "For more information about <topic>," unless it is really necessary. Otherwise, the link is sufficient.